pep8.py

```diff
@@ -268,6 +270,9 @@ def blank_lines(logical_line, blank_lines, indent_level, line_number,
                 yield 0, "E301 expected 1 blank line, found 0"
         elif blank_before != 2:
             yield 0, "E302 expected 2 blank lines, found %d" % blank_before
+    elif (logical_line and not indent_level and blank_before != 2 and
+          previous_logical_toplevel.startswith(('def', 'class'))):
+        yield 0, "E305 expected 2 blank lines before, found %d" % blank_before
```

Comment:
Shouldn't that expect 2 blank lines **after** the `def` or `class`? Sure it's checking for the next logical line if before it there are two lines but only because the block above was a `def` or `class`.

Maybe also mention that it expects two lines after a method-level function or class (I know E302 doesn't mention that too but that may be improved too?).